themes/Arch-Colors/lxqt-panel.qss

```diff
@@ -43,7 +43,7 @@ LXQtPanelPlugin > QToolButton {
 Plugin > QToolButton:hover,
 Plugin > QWidget > QToolButton:hover,
 LXQtPanelPlugin > QToolButton:hover {
-    color: #0a0a2b;
+    background: #1899da;
```

Comment:
Golden rule: no background color without a foreground color, and conversely. 

I see that the rule wasn't followed in some places of this theme before your change either. Sooner or later, you'll see unreadable texts somewhere. It needs to be corrected.